ansible/micromosler.yml

```diff
@@ -45,18 +52,162 @@
         - { src: 'glance-registry.conf',   dst: '/etc/glance/glance-registry.conf'                        }
         - { src: 'heat.conf',              dst: '/etc/heat/heat.conf'                                     }
         - { src: 'local_settings',         dst: '/etc/openstack-dashboard/local_settings'                 }
-        - { src: 'openstack_helper.sh',    dst: '/root/bin/openstack_helper.sh'                           }
+        - { src: 'mysql_db.conf',          dst: '/tmp/openstack-db.sql'                                   }
 
-    - shell: chmod 0755 /root/bin/openstack_helper.sh
-    - name: Running the openstack helper
-      shell: /root/bin/openstack_helper.sh  2>/tmp/oserr >/tmp/osout
+    # - file: path=/root/bin state=directory
+    # - copy: src={{ mm_home }}/configs/openstack_helper.sh dest=/root/bin/openstack_helper.sh
+    # - shell: chmod 0755 /root/bin/openstack_helper.sh
+    # - name: Running the openstack helper
+    #   shell: /root/bin/openstack_helper.sh  2>/tmp/oserr >/tmp/osout
 
-    - name: Removing any old image from Glance
-      shell: . /root/.keystonerc && (glance image-list | awk '/ {{ item }} /{print $2}' | while read img; do glance image-delete $img; done)
+    - name: Starting the Message Broker
+      service: name=rabbitmq-server state=started
+    - name: Preparing the RabbitMQ user
+      rabbitmq_user: state=present force=yes user=openstack password=rabbit configure_priv='.*' read_priv='.*' write_priv='.*'
+    # - shell: rabbitmqctl add_user openstack rabbit
+    #   ignore_errors: true
+    # - shell: rabbitmqctl set_permissions openstack ".*" ".*" ".*"
+    #   ignore_errors: true
+
+    - name: Stopping Openstack Services
+      service: name={{ item }} state=stopped
       with_items:
-        - project-computenode-stable 
-        - project-loginnode-stable
-        - topolino-q-stable
```

Comment:
Euh....sorry...I'm an idiot. I forgot one important thing: Other tenants will have their own set of machines. So we're good for that ;)

Just need to rethink the floating IPs...